serve/mlc_serve/engine/staging_engine.py

```diff
@@ -209,6 +211,7 @@ def step(self) -> InferenceStepResult:
 
                 with structlog.contextvars.bound_contextvars(**state.contextvars):
                     if seq_output.error is not None:
+                        LOG.info("Error")
```

Comment:
better logging here, if seq_output.error is an exception, try:

```suggestion
                        LOG.exception("An error occurred during generating sequence outputs.", exc=seq_output.error)
```

We should print some relevant information in the log here.